src/main/java/com/minecraftabnormals/abnormals_core/common/world/gen/ocean/OceanType.java

```diff
@@ -0,0 +1,8 @@
+package com.minecraftabnormals.abnormals_core.common.world.gen.ocean;
+
+/**
+ * The 5 different ocean types that generate in the world.
+ */
+public enum OceanType {
```

Comment:
I think it would be cleaner to have this enum be inside the `BiomeUtil` class to have less clutter.